beacon_node/beacon_processor/src/work_reprocessing_queue.rs

```diff
@@ -361,7 +361,12 @@ pub fn spawn_reprocess_scheduler<S: SlotClock + 'static>(
     executor: &TaskExecutor,
     slot_clock: S,
     log: Logger,
-) {
+    maximum_gossip_clock_disparity: Duration,
+) -> Result<(), String> {
+    // Sanity check
+    if ADDITIONAL_QUEUED_BLOCK_DELAY > maximum_gossip_clock_disparity {
```

Comment:
```suggestion
    if ADDITIONAL_QUEUED_BLOCK_DELAY >= maximum_gossip_clock_disparity {
```

I'm not sure if it *really* matters, but we need `>=` to make it equivalent to the previous assertion (below).

```rust
        assert!(
            beacon_processor::work_reprocessing_queue::ADDITIONAL_QUEUED_BLOCK_DELAY
                < beacon_chain::MAXIMUM_GOSSIP_CLOCK_DISPARITY
        );
```